src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -267,8 +281,9 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     if (!this._loadPromise) {
       this._loadPromise = new Promise((resolve, reject) => {
         this._lastTimeUpdate = startTime || 0;
+        const playbackStartTime = this._startTimeAttach || startTime || 0;
```

Comment:
use `this. _lastTimeDetach`. `this. _startTimeAttach` is unnecessary.

Reply:
Should keep the `this. _startTimeAttach` first cause if someone enter it as config it'll return always to this position